website/content/docs/overview/modules/index.md

```diff
@@ -6,120 +6,60 @@ weight: 40
-Here we describe what modules are and what default modules are installed using the default {{< name >}} installation.
+This page describes what modules are and how they are leveraged by the control plane to build the data plane flow.  
 
-# What are modules?
+As described in the [Architecture]({{< baseurl >}}/docs/overview/architecture/) page, the control plane generates a description of a data plane based on policies and application requirements. This is known as a blueprint, and includes components that are deployed by the control plane to fulfill different data-centric requirements.  For example, a component that can mask data can be used to enforce a data masking policy, or a component that copies data may be used to create a local data copy to meet performace requirements, etc. Modules are the way to describe such data plane components and make them available to the control plane. 
 
-At runtime, the control plane builds a blueprint for each application. The blueprint includes additional components that are injected into the data path by the control plane to fulfill governance requirements and application requirements.
+The functionality described by the module may be deployed (a) per workload, or (b) it may be a component that runs independent of the workload and its associated control plane.  In the case of (a), the Mesh for Data control plane handles the deployment of the functional component. In the case of (b) where the functionality of the module runs independently and handles requests from multiple workloads, a client module is what is deployed by the Mesh for Data control plane.  This client module passes parameters to the external component and monitors that status and results of the request to the external component.  The external component is declared as a dependency in the module yaml.
 
-Modules are the way to provide a specification of the capabilities of such components, and make them available to the control plane.
+In both cases, the module component is packaged as a [Helm](https://helm.sh/) chart that the control plane can install to a workload's data plane. To make a module available to the control plane it must be registered by applying a [`M4DModule`]({{< baseurl >}}/docs/reference/api/generated/app/#k8s-api-github-com-ibm-the-mesh-for-data-manager-apis-app-v1alpha1-m4dmodule) CRD.
 
-# Concept
+The following diagram shows an example with an Arrow Flight module that is fully deployed by the control plane and a second module where the client is deployed by the control plane but the ETL component providing the functionality has been independently deployed and supports multiple workloads.
 
-For the control plane to build a blueprint that includes modules, the control plane needs a catalog of all the modules available for its use.  Therefore, the control plane includes a **module repository** listing all registered modules. 
+![Example](ModuleArch2.png)
```

Comment:
That diagram may become outdated quickly. Please consider cutting it to the bottom left corner (taking M4DBlueprint Controller and below) and replacing "M4DBlueprint Controller" with "Control plane".

Reply:
If it's OK with you I would prefer to leave it for now.  Will update as things change.